pom.xml

```diff
@@ -708,5 +708,6 @@
         <module>molgenis-data-cache</module>
         <module>molgenis-data-platform</module>
         <module>molgenis-annotators-cmd</module>
+		<module>molgenis-mail</module>
```

Comment:
This PR will silently change your working mail server to a non-working mail server since username/password are not migrated from molgenis-server.properties. As an admin you might not notice this until users start telling you that they cannot register, change password etc.

How about extending MailSettings with a class that returns the corresponding property from molgenis-server.properties in case the value in the database is null? We could ditch this class later when we have a way to properly initialize the system.